po/README.md

```diff
@@ -0,0 +1,21 @@
+# Translations generated by Launchpad
+
+## How to sync translations from Launchpad?
+- Run `update-pot` to generate latest translation template (`po/snappy.pot`)
+- Import generated translation template to snappy entry [here](https://translations.launchpad.net/snapd/trunk/+templates)
+- Wait until import is approved. You can monitor status in the import queue [here](https://translations.launchpad.net/snapd/trunk/+imports)
+- Export latest translations [here](https://translations.launchpad.net/snapd/trunk/+export) (download link will be sent to your email)
+- Extract archive: `tar -xvf launchpad-export.tar.gz`
+- Create new directory to store processed translations `mkdir processed`
+- `for file in snappy-*; do msgattrib --no-obsolete --translated --output-file="processed/${file#snappy-}" "$file"; done;`
+    - Removes the `snappy-` prefix
+    - `--no-obsolete` removes obsolete #~ messages
+    - `--translated` excludes untranslated messages
+- Replace old translations under `/po` with new translations
+- Commit author and message should match Launchpad automated commits
+    - git commit --author="Launchpad Translations on behalf of snappy-dev[bot]<noreply@canonical.com>" -m "po: launchpad automatic translations update"
```

Comment:
Doesn't this clash against the cla check?

Reply:
Good point, I will remove this line for now. The future plan is to actually create a bot to do this periodically.